tsconfig.eslint.json

```diff
@@ -0,0 +1,13 @@
+// This config is only used to allow ESLint to use a different include / exclude setting than the actual build
```

Comment:
Does this file work? slash comments aren't valid JSON

Reply:
It was copy/auto generated, doesn't throw any error, so I assume it works?